src/components/PageHeadingDeployment.vue

```diff
@@ -17,10 +17,11 @@
   import { DeploymentMenu, DeploymentToggle, PageHeading, RunMenu } from '@/components'
   import { useWorkspaceRoutes } from '@/compositions'
   import { useCan } from '@/compositions/useCan'
-  import { Deployment } from '@/models'
+  import { Deployment, Flow } from '@/models'
 
   const props = defineProps<{
     deployment: Deployment,
+    flow: Flow,
```

Comment:
This works but means this is a breaking change. What we'd generally do is grab the flowId from the deployment and fetch it right within this component like `const { flow } = useFlow(() => props.deployment.flowId)`. 

That would also make this not a breaking change.